avocado/core/test.py

```diff
@@ -383,6 +384,11 @@ def _start_logging(self):
                                                        formatter,
                                                        self._ssh_logfile)
 
+        if isinstance(sys.stdout, output.LoggingFile):
+            sys.stdout.add_logger(logging.getLogger("avocado.test.stdout"))
+        if isinstance(sys.stderr, output.LoggingFile):
+            sys.stderr.add_logger(logging.getLogger("avocado.test.stderr"))
+
```

Comment:
OK, it is used in case of failure: https://travis-ci.org/avocado-framework/avocado/jobs/178772163#L1749

anyway the first patch seems to be sufficient to avoid this issue. In fact it does the same as the second patch, it ignores the writes until the logger is configured. So I think it'd be better to keep the initialization and add `continue` into writes if there is nowhere to write to. What do you think about this?